Alanoud_AlRasheed/week_06/day_01/ActorApp/app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>ActorApp</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+  <nav>
+ <span> <%= link_to 'All Actors', actors_path %> </span>
+<span><%= link_to 'Add Actor', new_actor_path %> </span>
+<span><%= link_to 'All Movies', movies_path %> </span>
+<span><%= link_to 'Add Movie', new_movie_path %> </span>
```

Comment:
This works perfectly, but it might be worthwhile looking into the `partial` approach as it gives you more flexibility in terms of picking pages to add navs. Just a thought!